website/docs/sidebar.md

```diff
@@ -26,7 +26,18 @@ module.exports = {
 
 ## Sidebar object
 
-A sidebar object looks like the following. The key `docs` is the name of the sidebar (can be renamed to something else) and `Getting Started` is a category within the sidebar. `greeting` and `doc1` is just a [Sidebar Item](#sidebar-item).
+A sidebar object is defined like this.
+
+```typescript
+type Sidebar = {
+  [sidebarId: string]: {
+    [sidebarCategory: string]: SidebarItem[];
+  } | SidebarItem[];
+}
+```
+
+Below is an example of a sidebar object. The key `docs` is the id of the sidebar (can be renamed to something else) and `Getting Started` is a category within the sidebar. `greeting` and `doc1` is just a [Sidebar Item](#sidebar-item).
```

Comment:
grammar: greeting and doc1 -are- sidebar items.

(im on a mobile device so not convenient to commit directly)